__tests__/rtFixtureLoaderHelper.test.js

```diff
@@ -2,13 +2,13 @@
 
```

Comment:
change name to `fixtureLoaderHelper.test.js`

Reply:
will do.